srcpkgs/hobbits/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'hobbits'
+pkgname=hobbits
+version=0.52.0
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-host-tools qt5-qmake"
+makedepends="fftw-devel libusb-devel libpcap-devel python3-devel qt5-devel"
+short_desc="Multi-platform GUI for bit-based analysis, processing and visualization"
+maintainer="classabbyamp <dev@kb6.ee>"
+license="MIT, GPL-2.0-or-later"
+homepage="https://mahlet-inc.github.io"
+distfiles="https://github.com/Mahlet-Inc/hobbits/archive/refs/tags/v${version}.tar.gz"
+checksum=3332e02380ab46649f0ec2066b63b6e0fb04b55cdca973d28880f1607ebbf27e
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" python3"
```

Comment:
Does this need to be conditional?  Couldn't it just go in `hostmakedepends` even for native?

Reply:
it works fine either way, so I guess it could be non-conditional. I've updated the template to do so.